src/main/java/org/traccar/reports/Summary.java

```diff
@@ -103,9 +106,21 @@ public static void getExcel(OutputStream outputStream,
             Date from, Date to) throws SQLException, IOException {
         ReportUtils.checkPeriodLimit(from, to);
         Collection<SummaryReport> summaries = getObjects(userId, deviceIds, groupIds, from, to);
-        String templatePath = Context.getConfig().getString("report.templatesPath",
-                "templates/export/");
-        try (InputStream inputStream = new FileInputStream(templatePath + "/summary.xlsx")) {
+
+        String templatePath;
+        Path filePath= Paths.get("templates/export/" + "/summary_" + userId + ".xlsx");
+        if(Files.exists(filePath))
+        {
+             templatePath = Context.getConfig().getString("report.templatesPath",
+                    "templates/export/" + "/summary_" + userId + ".xlsx");
+        }
+        else
+        {
+            templatePath = Context.getConfig().getString("report.templatesPath",
+                    "templates/export/" + "/summary.xlsx");
+        }
```

Comment:
Use ternary operator or fix code style errors:
- one empty line before if statement (112)
- space after if and bracket (line 112)
- else should be in line with brackets (  } else { )

As we are working in existing project, we have to follow its code style and practices.